docs/ops/deployment/docker.md

```diff
@@ -246,10 +246,15 @@ At the end you can call the standard `/docker-entrypoint.sh` script of the Flink
     # create custom_plugin.jar
 
     echo "
-    ln -fs /opt/flink/opt/flink-metrics-graphite-*.jar /opt/flink/lib/.  # enable an optional library
+    ln -fs /opt/flink/opt/flink-queryable-state-runtime-*.jar /opt/flink/lib/.  # enable an optional library
     ln -fs /mnt/custom_lib.jar /opt/flink/lib/.  # enable a custom library
-    ln -fs /opt/flink/opt/flink-s3-fs-hadoop-*.jar /opt/flink/lib/.  # enable an optional plugin
-    ln -fs /mnt/custom_plugin.jar /opt/flink/plugins/.  # enable a custom plugin
+
+    mkdir -p /opt/flink/plugins/flink-s3-fs-hadoop
```

Comment:
well that's inconvenient isn't it :/ Nothing we can fix though on this end.

Reply:
not atm as I understand, it is basically copied from the `plugins` how-to.
we should have packed it respectively in `/opt/flink/opt` which is another issue.